src/components/GroceryItem.svelte

```diff
@@ -0,0 +1,155 @@
+<script>
+  export let item;
+  export let removeItem;
+  export let updateItem;
+  export let editItem;
+  export let itemId;
+  export let isEditing = false;
+
+  export let inputEl;
+  let trashIconImage = "./images/trash.svg";
+  let editIconImage = "./images/edit-icon.svg";
+  //export let inputEl = item.name;
+</script>
+
+<li class="item">
+  {#if isEditing}
+    <input class="input-update" type="text" bind:value={inputEl} />
+    <button
+      class="update-btn"
+      on:click={() => {
+        updateItem(itemId, inputEl);
+      }}>Update</button
+    >
+  {:else}
+    <!--<span class="name">{item.name}</span>
+  <button class="edit-item-btn btn-white btn-animate"
+  on:click={() => {
+      isEditing=true;
+  }}><img src={editIconImage} alt="{editIconImage} image" /></button>-->
+
+    <span class="name">{item.name}</span>
+    <button
+      class="edit-item-btn btn-white btn-animate"
+      on:click={() => {
+        editItem();
```

Comment:
When one item is set as `isEditing`, all the items enter in that mode. Didn't you prefer the "isEditing" to be individual per item?

Reply:
@sandrina-p Yes I think It'd be more relevant.